docs/sequence/PISPTransactionApi.puml

```diff
@@ -71,78 +76,214 @@ rnote right of PISP #Light
   "amountType": "SEND"
 }
 end note
-PISP -> OBS: ""POST /thirdpartyTransactions/{id}/initiate""
+PISP -> 3p_PISP_OBS: POST /thirdpartyTransactions/{id}/initiate
 activate PISP
-activate OBS
-' TODO: line up with state-models.md doc
+activate 3p_PISP_OBS
 
-OBS -> ST8: ""initiate()""
+3p_PISP_OBS -> ST8: initiate()
 activate ST8
-rnote right of ST8: ""state: requestTransaction""
-
-ST8 -> Switch: ""POST /thirdpartyRequests/transactions""
-Switch --> ST8: ""202 Accepted""
-rnote right of ST8: ""state: pendingTransactionAuthorization""
+rnote right of ST8: state: requestTransaction
 
+ST8 -> Switch: POST /thirdpartyRequests/transactions
+activate Switch
+Switch --> ST8: 202 Accepted
+deactivate ST8
+rnote right of ST8: state: pendingTransactionInitiation
 
-note over Switch
-  Switch kicks off the transaction - DFSP issues ""POST /authorizations""
+note over 3p_DFSP_IBS
+  DFSP handles PISP transaction request
 end note
+Switch -> 3p_DFSP_IBS: POST /thirdpartyRequests/transactions
+activate 3p_DFSP_IBS
+3p_DFSP_IBS --> Switch: 202 Accepted
+rnote right of 3p_DFSP_IBS: state: transactionRequestReceived
+note over 3p_DFSP_IBS
+  **transactionRequestState** is **RECEIVED**
+end note
+deactivate Switch
+3p_DFSP_IBS -> Switch: GET /consents/{consentId}
```

Comment:
I'm not sure I understand this call here - the consent should always be known by the DFSP

Reply:
I suppose we might want to talk to the auth service here, but I'd say we can rely on the DFSP also knowing about the consent here.

Reply:
from which part of the context for this workflow DFSP will derive about `Consent`? Currently we are getting `consentId` in the payload: https://github.com/mojaloop/thirdparty-api-adapter/blob/master/src/interface/api-template.yaml#L806-L863, so DFSP needs to get details about - in our scenario DFSP retrieves it from AUTH-SERVICE

Reply:
I don't think that's right. I think the DFSP needs to keep a record of the `consentId`, as well as (optionally) storing it with the Auth-Service. Let's discuss more after the standup.